### PR TITLE
[5.5] Apply clean code concepts

### PR DESCRIPTION
**Pull request by joaorobertopb**
Apply clean code concepts:

- remove unnecessary else statement
- use ternary operator

### CODE REVIEW

File: src/Oci8/Eloquent/OracleEloquent.php [hunk 1]
```diff
@@ -293,11 +293,9 @@ protected function performInsert(Builder $query)
      */
     protected function insertAndSetId(Builder $query, $attributes)
     {
-        if ($binaries = $this->extractBinaries($attributes)) {
-            $id = $query->getQuery()->insertLob($attributes, $binaries, $keyName = $this->getKeyName());
-        } else {
-            $id = $query->insertGetId($attributes, $keyName = $this->getKeyName());
-        }
+        $id = ($binaries = $this->extractBinaries($attributes))  ?
```
Comment (yajra) [inline src/Oci8/Eloquent/OracleEloquent.php]: Please remove the excess space `  ?` to pass on styleci.

**Comment by yajra** [comment]
Looks good to me. Please fix the cs and I think this is good to merge. Thanks!

**Comment by joaorobertopb** [comment]
Thanks!

**Comment by yajra** [comment]
Will tag this when PR #340 is released and applied to on 5.5. Thanks!

**Pull request merged**